ImprovedAnvil/lib/optional/all_protagonists_extra_rewards.TPH

```diff
@@ -0,0 +1,36 @@
+// Remove class restrictions from nonessential kiit drios
+
+// Golden Girdle in Watcher's Keep level 4
+COPY_EXISTING ~override\AR3017.BCS~ ~override\AR3017.BCS~
+  DECOMPILE_AND_PATCH BEGIN
+    REPLACE_TEXTUALLY ~!Class(Player1,MAGE_ALL)~ ~~
+  END
+
+// Scroll of Memory Boosting in Spellhold
```

Comment:
these drops are protagonist-based treasure: vagrant gets cloak of magic shielding, mages and druids - a scroll of memory boosting, the rest get a barbarian essence potion.
in its current form this block messes stuff up. I'm dropping it for a moment. Let me know what you want to do here, and I'll reimplement it properly.

Reply:
Nevermind, I know what to do here

Reply:
i'm not sure what you did but i changed the variable name for the scroll in the non-essential loot addon, so in principle the party would get both the cloak and scroll, and the barbarian essence potion drop essentially never happens.  that seemed to make sense to me - what did you end up doing?